CHANGELOG.md

```diff
@@ -8,6 +20,10 @@
 
 - Fixes [\#37761](https://projects.theforeman.org/issues/37761) - use ProxyPass and upgrade=websocket where possible [\#1185](https://github.com/theforeman/puppet-foreman/pull/1185) ([evgeni](https://github.com/evgeni))
 
+## [24.2.1](https://github.com/theforeman/puppet-foreman/tree/24.2.1) (2024-09-04)
```

Comment:
how did that happen? because you maintain multiple branches at the same time?

Reply:
Yes, it gets really confused by that.